e2e_deployment_files/stress_deployment.template.json

```diff
@@ -168,6 +168,18 @@
               },
               "ConsumerGroupId": {
                 "value": "<Analyzer.ConsumerGroupId>"
+              }, 
+              "LogAnalyticEnabled": {
+                "value": "<Analyzer.LogAnalyticEnabled>"
+              },
+              "LogAnalyticWorkspaceId": {
```

Comment:
just want to confirm that if LogicAnalyticEnabled is set to "false" and other parameters are not provided in e2e test script, does LogAnalyticWorkspaceId, LogAnalyticSharedKey and LogAnalyticLogType update to empty string?

Reply:
correct